Automattic/woocommerce-payments
# Translatable Strings for Business type, description and structure

Fixes #4026 

#### Changes proposed in this Pull Request

Adds Translatable Strings for Business Type, Business Description and Business Structure.

#### Testing instructions

* Go to the Payments > Onboarding page.
* Try changing between various business types/countries and verify the dropdown options for business type and business structure display the appropriate strings.
* *For business types*: The US is currently the only country which has unique descriptions for the business types. (All others are generic)
* *For business structures*: AU, HK, SG, NZ and US all have unique business structures.
-------------------


**Post merge**
